mappings/net/minecraft/block/BeeHiveBlock.mapping

```diff
@@ -1,9 +1,34 @@
 CLASS net/minecraft/class_4481 net/minecraft/block/BeeHiveBlock
+	FIELD field_20418 NATRURAL_FACINGS [Lnet/minecraft/class_2350;
```

Comment:
"natrual"? Should it be "natural", and if yes, I'm not sure what "natural facings" means...

Reply:
like the possible faces for world gen

Reply:
Maybe `VALID_FACINGS` would be better. It's not just worldgen, you also can't place a bee hive facing up.

Reply:
it won't face north for worldgen, only south east west

Reply:
Ok then.